.gitignore

```diff
@@ -60,4 +60,5 @@ build/mage_output_file.go
 !**/**/testdata/*.rego
 
 release-notes.md
-config/*.*.bk
\ No newline at end of file
+config/*.*.bk
+config/dev.yml
```

Comment:
i got tired of remembering to not commit my changed `config/local.yml` so i decided to create `config/dev.yml` and add it to .gitignore

the mage:dev task will now look for this file first and fallback to `config/local.yml` if not found